File: C++17/백준/Bronze/11098. 첼시를 도와줘！/첼시를 도와줘！.cc
#include <iostream>
#include <string>
using namespace std;

int main (void) {
    int n;
    cin >> n;
    
    for (int i = 0; i < n; i++) {
        int p;
        cin >> p;
        
        string top_plr = "";
        long long max_price = -1;
        
        for (int j = 0; j < p; j++) {
            long long price;
            string name;
            cin >> price >> name;
            
            if (price > max_price) {
                max_price = price;
                top_plr = name;
            }
        }
        cout << top_plr << endl;
    }
    
    return 0;
}

File: Java/백준/Bronze/11098. 첼시를 도와줘！/첼시를 도와줘！.java
import java.util.Scanner;

public class Main {
    public static void main (String[] args) {
        Scanner sc = new Scanner (System.in);
        
        int n = sc.nextInt();
        
        for (int i = 0; i < n; i++) {
            int p = sc.nextInt();
            
            String top_plr = "";
            long max_price = -1;
            
            for (int j = 0; j < p; j++) {
                long price = sc.nextLong();
                String name = sc.next();
                
                if (price > max_price) {
                    max_price = price;
                    top_plr = name;
                }
            }
            System.out.println(top_plr);
        }
        
        sc.close();
    }
}